pkg/validation/release.go

```diff
@@ -18,6 +18,9 @@ func validateReleases(fieldRoot string, releases map[string]api.UnresolvedReleas
 		names.Insert(name)
 	}
 	for _, name := range names.List() {
+		if err := partOfImageStreamName(name); err != nil {
+			validationErrors = append(validationErrors, fmt.Errorf("%s[%s]: the release name is not a valid part of an image stream name: %w", fieldRoot, name, err))
```

Comment:
Users generally don't know the implementation details of release image streams, so this is too specific.  Just saying it's invalid because it has a period should be enough.

Reply:
Done.